e2e/client/specs/helpers/monitoring.ts

```diff
@@ -396,7 +396,7 @@ class Monitoring {
         this.tabAction = function(tab) {
             const btn = element(by.css('[ng-click="vm.current_tab = \'' + tab + '\'"]'));
 
-            browser.wait(ECE.elementToBeClickable(btn), 2000);
+            browser.wait(ECE.elementToBeClickable(btn));
```

Comment:
this will make it wait for minutes before failing, not sure that's useful in general